### PR TITLE
Add deploy scripts for clingo-lp

### PR DESCRIPTION
**Pull request by rkaminsk**
# TODO

- [x] add pip source package
  - [ ] <strike>add lpsolve as dependency</strike>
- [ ] <strike>add ppa package</strike>
- [x] add conda package

# Notes

I could only add a pip and conda packages due to the limited availability of lpsolve. There is no usable pip package for lpsolve. Users will have to install lpsolve or cplex manually when using pip. I did not add either of them as dependencies. I also could not provide a Ubuntu PPA package because the lp-solve package does not provide Python bindings. Conda is probably the best way to install and use clingo-lp.

In summary, we can now trigger deploys for pip and conda via github actions.

### CODE REVIEW

**Comment by sthiele** [comment]
Can you confirm what I think is the workflow to make a new release:
1. bump the version in setup.py
2. trigger deploy scripts.
3. check if the packages work.
4. (optional) create a release tag on github

**Comment by rkaminsk** [comment]
> Can you confirm what I think is the workflow to make a new release:
> 
>     1. bump the version in setup.py

I always bump the (patch) version right after the release. Like this, development packages always have a higher version than the last release. Further adjustments are of course possible at any time.

>     2. trigger deploy scripts.

The deploy scripts can trigger both development and release scripts. I always test the development builds first. The deploy script can then simply be run with the revision to release.

>     3. check if the packages work.

Obviously.

>     4. (optional) create a release tag on github

Yup.

**Comment by sthiele** [comment]
Thanks!
